### ISSUE TITLE
FR: Add user definable icons when adding New Host

### ISSUE THREAD

**Issue by tjsuominen** [basildane/WakeOnLAN#26]
For easy visual detection off hosts, it would be great to be able to add (import) user selectable images (as icons) e.g. in .PNG-format when defining New hosts (optionally, of course).

**Comment by basildane** [comment]
Good idea.

**Comment by tjsuominen** [comment]
Any news @basildane on implementing this? Or providing more built-in icons to choose from?

**Comment by basildane** [comment]
This is a significant change and I just don't have any time right now.

**Comment by tjsuominen** [comment]
That's a fair statement. Then just keep focus on more essential development stuff. Still, I'd say it would be a great UX thing to have in the future.

**Comment by jpdonato** [comment]
I agree that the possibility of changing a client's icon in the GUI would be most interesting. This would expand the program's usefulness to devices that don't even support WOL per se: one could enter all devices on their network requiring monitoring or occasional remote logging - like printers, switches, routers, etc. At a glance, the device type and status could be determined from the icon, and any remote logins could easily be initiated as desired from the GUI (no need to change the source code there since WOL now supports this), for these other networked devices. Cheers.
